packages/core/r4b/data-types/boolean.ts

```diff
@@ -0,0 +1,88 @@
+import { merge } from "lodash";
```

Comment:
This one is tricky - lodash is not an esm module, so to ensure compatiblity you need to import it from the root only.

```suggestion
import _ from "lodash";
```